### PR TITLE
[SDK-1057] Updated iOS Push Story documentation

### PR DESCRIPTION
**Pull request by lowip**
# Pull Request/Issue Resolution

**Description of Change:**

Updating iOS Push Story documentation to reflect upcoming changes in integration steps.

Closes [SDK-1057](https://jira.braze.com/browse/SDK-1057)

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__release date yet to be decided__)
- [x] No

---
---

<details>
<summary>PR Checklist</summary>

- [x] Check that all links work!
- [x] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [x] Tag others as Reviewers as necessary.
- [x] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

**Review by davidbielik — APPROVED** [review]

**Review by Bucimis — APPROVED** [review]

**Review by chshapiro — DISMISSED** [review]

File: _docs/_developer_guide/platform_integration_guides/ios/push_story.md [hunk 1]
```diff
@@ -47,73 +47,41 @@ You also need to add Capability `App Groups`. If you haven't had any app group i
...
 target 'YourContentExtensionTarget' do
-  pod 'Appboy-Push-Story', '~>3.0'
+  pod 'Appboy-Push-Story'
 end
 ```
 
 After updating the Podfile, navigate to the directory of your Xcode app project within your terminal and run `pod install`
 
-## Step 5: Link the Braze Push Story Framework
+### Manually
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/push_story.md:72]: Small nit:
```suggestion
### Manual Integration
```

**Review by hokstuff — DISMISSED** [review]
LGTM

**Comment by lowip** [comment]
Updated with latest develop. This is now ready to be merged.

**Review by KellieHawks — APPROVED** [review]
lgtm!

**Pull request merged**